packages/config/config/devices/0x0234/templates/logic_group_template.json

```diff
@@ -0,0 +1,560 @@
+{
+	"enable_central_scene": {
+		"$import": "../../templates/master_template.json#base_enable_disable",
+		"label": "Central Scene",
+		"defaultValue": 1
+	},
+	"multilevel_enable": {
+		"$import": "../../templates/master_template.json#base_enable_disable",
+		"valueSize": 4,
+		"defaultValue": 1
+	},
+	"multilevel_upper_function": {
+		"$import": "../../templates/master_template.json#base_0-99_nounit",
+		"label": "Upper switch value",
+		"valueSize": 4,
+		"defaultValue": 255
+	},
+	"multilevel_lower_function": {
+		"$import": "../../templates/master_template.json#base_0-99_nounit",
+		"label": "Lower switch value",
+		"valueSize": 4,
+		"defaultValue": 0
+	},
+	"multilevel_duration_function": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"label": "Duration",
+		"valueSize": 4
+	},
+	"multilevel_disabled_byte": {
+		"label": "Disable",
```

Comment:
What does this parameter do? Just "Disable" is a bit unspecific.

Reply:
Correct, the byte is disabled. 
![image](https://user-images.githubusercontent.com/33527165/114500773-0962e380-9c29-11eb-90c4-68229599498c.png)

Reply:
If the entire byte isn't used for anything, then just remove it from the files.